src/lib/containers/RenderIfInView.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
```

Comment:
This is my code.  This is the component that I'm going to export for portals (with the ability to force pre-render on the portal side).  Every dynamically generated SynapseComponent will be wrapped in one of these, unless you set some new flag (like forcePreload) to true (which could be done for complex components down the page that need extra time to load!).